nrfx/hal/nrf_nvmc.h

```diff
@@ -133,6 +134,44 @@ NRF_STATIC_INLINE void nrf_nvmc_nonsecure_mode_set(NRF_NVMC_Type *    p_reg,
                                                    nrf_nvmc_ns_mode_t mode);
 #endif
 
+/**
+ * @brief Function for writing a word into the flash
+ *
+ * Before calling this function, the caller must ensure that:
+ *  * The address is word aligned
+ *  * Write mode is enabled and
+ *  * The flash is ready to accept another write
+ *
+ * Users are recommended to use the NVMC driver
+ * nrfx_nvmc_word_write() instead.
+ *
+ * @param[in] addr  Address of the word to write
+ * @param[in] value Value to write
+ */
+NRF_STATIC_INLINE void nrf_nvmc_word_write(uint32_t addr,
+                                           uint32_t value);
+
+/**
+ * @brief Function for reading a word from the flash
+ *
+ * @param[in] addr  Address of the word to read
```

Comment:
too much spaces after `addr`.

Also, we could use `address` instead, to better align with nrfx_nvmc.h naming scheme